terraform/scripts/set-up-pingdom.sh

```diff
@@ -4,7 +4,7 @@ set -eu
 TERRAFORM_ACTION=${1}
 VERSION=0.2.2
 BINARY=terraform-provider-pingdom-tf-0.8.5-$(uname -s)-$(uname -m)
-STATEFILE=pingdom-${MAKEFILE_ENV_TARGET}.tfstate
```

Comment:
Context for why we originally used `MAKEFILE_ENV_TARGET` is described in b8158b24b29ef93db67c6bcd5872434a4b8996e1. The use in this script contradicts that though 😸 

So I think it's fine to remove it. We have two places where configuration can change depending on the environment - `Makefile` or Terraform `tfvars` (as used in this script).